src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.Api/Startup.cs

```diff
@@ -90,7 +90,7 @@ public void ConfigureServices(IServiceCollection services)
             }
             
             services.AddApplicationInsightsTelemetry(_configuration["APPINSIGHTS_INSTRUMENTATIONKEY"]);
-            services.AddOpenTelemetryRegistration(_configuration["APPLICATIONINSIGHTS_CONNECTION_STRING"]);
+            //services.AddOpenTelemetryRegistration(_configuration["APPLICATIONINSIGHTS_CONNECTION_STRING"]);
```

Comment:
I think this needs to be uncommeted.
I reckon its worth while adding a stub implementation of the IMetrics so we have something like
```
    public class StubFindAnApprenticeshipMetrics : IMetrics
    {
        public void IncreaseVacancyViews(string vacancyReference, int viewCount = 1)
        {
        }
    }
```
and the extension is changed to:
```
public static class AddOpenTelemetryExtensions
    {
        /// <summary>
        /// Add the OpenTelemetry telemetry service to the application.
        /// </summary>
        /// <param name="services">Service Collection</param>
        /// <param name="appInsightsConnectionString">Azure app insights connection string.</param>
        public static void AddOpenTelemetryRegistration(this IServiceCollection services, string appInsightsConnectionString)
        {
            if (!string.IsNullOrEmpty(appInsightsConnectionString))
            {
                // This service will collect and send telemetry data to Azure Monitor.
                services.AddOpenTelemetry().UseAzureMonitor(options =>
                    {
                        options.ConnectionString = appInsightsConnectionString;
                    })
                    // Configure metrics
                    .WithMetrics(opts => opts
                        .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
                            Constants.OpenTelemetry.ServiceName,
                            nameof(FindAnApprenticeship)))
                        .AddMeter(Constants.OpenTelemetry.ServiceMeterName));
                services.AddSingleton<IMetrics, FindAnApprenticeshipMetrics>();
            }
            else
            {
                services.AddSingleton<IMetrics, StubFindAnApprenticeshipMetrics>();
            }
        }
    }
    ```